scripts/lib/cps_db_stunnel_manager.py

```diff
@@ -313,7 +313,7 @@ def sig_handler(signum, frame):
     cps.obj_register(handle, cps.key_from_name("target","cps/db-instance"), db_cb)
 
     # WARNING systemd module not currently installed!!!
-    #         Waiting for legal approval to bring systemd module into OS10.
+    #         Waiting for legal approval to bring systemd module into OPX.
```

Comment:
@ArthiSivanantham, unlike OS10, there is no legal hurdle for using python-systemd in OPX.  Perhaps we should just do it now.  At the very least, the comment is confusing in the OPX context.